yarn.config.cjs

```diff
@@ -159,7 +159,7 @@ module.exports = defineConfig({
       if (isChildWorkspace) {
         // The list of files included in all non-root packages must only include
         // files generated during the build process.
-        expectWorkspaceField(workspace, 'files', ['dist/']);
+        // expectWorkspaceField(workspace, 'files', ['dist/']);
```

Comment:
Not ideal. Will need to discuss with the team a better way to handle the "files" field in packages.

To avoid module resolution issues (see [prev comment](https://github.com/MetaMask/core/pull/4604/files#r1717043427)), we are exposing this folder paths.

However for them to be included in the module, we need to add then to the `files` field of the `package.json`- [see example](https://github.com/MetaMask/core/pull/4604/files#diff-7855eaf5406cbf1191aed59d02f2fd928544f17eefcf527a8d0feb7811032b5fR55-R57)

Reply:
Created a separate PR that will better resolve this issue.
https://github.com/MetaMask/core/pull/4611

Reply:
Awesome https://github.com/MetaMask/core/pull/4611 has been merged, will resolve conflicts in this PR.